<?xml version="1.0" encoding="UTF-8"?>
<bajadoc version="2.0" createdBy="niagara-baja-doclet-1.0.9" createdAt="03-May-2024" createdOn="883e7f7a9875">
<class module="export" runtimeProfile="rt" qualifiedName="javax.baja.export.ExportException" name="ExportException" packageName="javax.baja.export" public="true" category="exception">
<description>
Something went wrong in the export process.&#xa; &lt;p&gt;&#xa; API Status: **Development**
</description>
<tag name="@since">Niagara 4.8</tag>
<extends>
<type class="javax.baja.sys.BajaException"/>
</extends>
<!-- javax.baja.export.ExportException(java.lang.String) -->
<constructor name="ExportException" public="true">
<parameter name="message">
<type class="java.lang.String"/>
</parameter>
<description/>
</constructor>

<!-- javax.baja.export.ExportException(java.lang.Throwable) -->
<constructor name="ExportException" public="true">
<parameter name="cause">
<type class="java.lang.Throwable"/>
</parameter>
<description/>
</constructor>

</class>
</bajadoc>
